ddtrace/internal/module.py

```diff
@@ -145,9 +146,10 @@ def find_loader(fullname):
 
 
 class _ImportHookChainedLoader(Loader):
-    def __init__(self, loader):
-        # type: (Loader) -> None
+    def __init__(self, loader, module_spec=None):
+        # type: (Loader, Optional[ModuleSpec]) -> None
         self.loader = loader
+        self.module_spec = module_spec
```

Comment:
Why is this added? Does it do something magical to `Loader` objects based on an implementation detail?

Reply:
You're right, I was misusing the `Loader` object there, the `Finder` is what I actually needed